addons/mail/static/src/core/common/message_model.js

```diff
@@ -80,6 +80,7 @@ export class Message extends Record {
     is_note;
     /** @type {boolean} */
     is_transient;
+    isEditing = false;
```

Comment:
This newly introduced field looks unused

Reply:
Woops, removed :+1: